provider-dropbox/src/java/org/sakaiproject/archiver/provider/DropboxArchiver.java

```diff
@@ -166,8 +166,7 @@ private String getFilename(final ContentResource resource) {
 	 */
 	private Map<String, String> getUserDisplayNames(final List<String> userUuids) {
 		final List<User> users = this.userDirectoryService.getUsers(userUuids);
-		final Map<String, String> userMap = users.stream().collect(Collectors.toMap(User::getId, User::getSortName));
-		return userMap;
+		return users.stream().collect(Collectors.toMap(User::getId, User::getSortName));
```

Comment:
this is  borderline, but since its a small method its ok as it is easy to see what the return type is. sonar sometimes goes overboard with the 'useless assignment' rule.